malaria24/ona/tests/test_models.py

```diff
@@ -496,6 +496,145 @@ def test_send_digest_email(self):
         self.assertEqual(html_content.count('EHP1, EHP2'), 10)
         self.assertEqual(message.to, [mis.email_address])
 
+    @responses.activate
+    def test_print_district_digest_email_data(self):
+        Facility.objects.create(facility_code='342315',
+                                facility_name='Facility 1',
+                                province='Limpopo',
+                                district=u'Example1')
+        Facility.objects.create(facility_code='343434',
+                                facility_name='Facility 2',
+                                province='Limpopo',
+                                district=u'Example2')
+        Facility.objects.create(facility_code='333444',
+                                facility_name='Facility 3',
+                                province='Limpopo',
+                                district=u'Example3')
+        Facility.objects.create(facility_code='333333',
+                                facility_name='Facility 3',
+                                province='The Eastern Cape',
+                                district=u'Example4')
+        Facility.objects.create(facility_code='111111',
+                                facility_name='Facility 1',
+                                province='Gauteng',
+                                district=u'Example5')
+        Facility.objects.create(facility_code='333333',
```

Comment:
I think you have this facility twice?
you probably don't need to create any of the facilities that you don't create cases for.